app/scripts/modules/core/src/presentation/formsy/components/Input.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+
+import { FormComponent, IFormComponentState, IFormComponentProps } from '../FormComponent';
+
+export interface IProps extends IFormComponentProps {
+  className: string;
+  label: string;
+  type: string;
+}
+
+export interface IState extends IFormComponentState { }
+
+/** A simple Formsy form component for validated <input> tags (text or checkbox) */
+export class Input extends FormComponent<string, IProps, IState> {
```

Comment:
I'm not actually using this component, but this shows a simpler approach which doesn't accept a layout strategy.